cmd/kubeadm/app/constants/constants.go

```diff
@@ -262,6 +262,10 @@ const (
 	// DefaultEtcdVersion indicates the default etcd version that kubeadm uses
 	DefaultEtcdVersion = "3.3.10"
 
+	// DefaultEtcdImageVersion indicates the etcd image version that kubeadm uses.
+	// For example, the image version of "k8s.gcr.io/etcd:3.3.15-0" is "0".
+	DefaultEtcdImageVersion = "0"
```

Comment:
should we call this `DefaultEtcdRevision`?

Reply:
You're right. We don't need this. We can just use "3.3.15-0" as the tag directly.